.travis.yml

```diff
@@ -110,6 +110,9 @@ script:
           cd _build;
       fi
     - $make update
+    - if [ -n "$CHECKDOCS" ]; then
+          $make doc-nits;
```

Comment:
Errr...  you either need to change this to `perl util/find-doc-nits.pl`, or to add a doc-nits target to `Configurations/unix-Makefile.tmpl`